### PR TITLE
Watersmart 631

### PR DESCRIPTION
**Pull request by ayan-usgs**
Load the database with characteristic data from the Yahara River.

### CODE REVIEW

File: nldi-liquibase/src/main/resources/liquibase/characteristic_data/tables.sql [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+,units                          character varying(4000)
+,dataset_label                  character varying(4000)
+,dataset_url                    character varying(4000)
+,theme_label                    character varying(4000)
+,theme_url                      character varying(4000)
+,characteristic_type            character varying(4000)
+,constraint characteristic_metadata_pk
+    primary key (characteristic_id)
+)
+WITH (OIDS=TRUE);
```
Comment (dsteinich) [inline nldi-liquibase/src/main/resources/liquibase/characteristic_data/tables.sql]: I'm not sure we want this option. A brief search makes it look like a legacy postgresql specific thing.

File: docker-compose.yml [hunk 2]
```diff
@@ -27,7 +27,7 @@ services:
     volumes_from: 
       - nldi_db_vol_container
     ports:
-      - "5432:5432"
+      - "5434:5432"
```
Comment (dsteinich) [inline docker-compose.yml]: The readme should be changed to reflect this.

File: nldi-liquibase/src/main/resources/liquibase/characteristic_data/tables.sql (hunk 1 above)
Comment (ayan-usgs) [inline nldi-liquibase/src/main/resources/liquibase/characteristic_data/tables.sql]: My understanding of it is that the syntax has changed, but OIDS are still something that is sanctioned for use. That said, I'm skeptical of there utility in this schema... the values are only four bytes in size and can't guarantee uniqueness once the database reaches a certain size and the values turnover.

I'm going to take it out.

File: docker-compose.yml (hunk 2 above)
Comment (ayan-usgs) [inline docker-compose.yml]: Thanks for catching that.

**Pull request merged**